content/blog/2022-05-04-e2e-cv-pt2.md

```diff
@@ -0,0 +1,197 @@
+  ├── 675efb3 [exp-6c274]   Apr 09, 2022      0.10356     0.069076          0.90321         0.75906      0.92371     0.71492    0.98099   0.1                   16                 770745a
+  └── c8b1857 [exp-04bcd]   Apr 09, 2022      0.11189     0.088695          0.86905         0.75296      0.92005     0.71619    0.98025   0.01                  8                  094c420
+ ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+```
+
+Once we identify one or a few best ones (e.g., highest `dice_mean` score), we
+can either
+[persist](https://dvc.org/doc/user-guide/experiment-management/persisting-experiments)
+them by creating a branch out of an experiment:
+
+```bash
+$ dvc exp branch exp-04bcd my-branch
+Git branch 'my-branch' has been created from experiment 'exp-04bcd'.
+To switch to the new branch run:
+        git checkout my-branch
+```
+
+or we can share them with our teammates by
+[uploading](https://dvc.org/doc/user-guide/experiment-management/sharing-experiments#uploading-experiments)
+them to our git repo:
+
+```bash
+$ dvc exp push origin exp-04bcd
+Pushed experiment 'exp-04bcd'to Git remote 'origin'.
+```
+
+## Collaboration and Reporting with Iterative Studio
+
+What if we needed to report the results to our team members or maybe hand over
+the project to one of them? How do we communicate everything we did since the
+conception of the project? What things resulted in the most significant
+improvements? What things didn't seem to matter at all?
+
+[Iterative Studio](https://studio.iterative.ai/) is a web-based application with
+seamless integration with DVC for data and model management, experiment
+tracking, visualization, and automation. It becomes especially valuable when
+collaborating with others on the same project or when there's a need to
+summarize the progress of the project through metrics and plots. All that's
+needed is to connect the project's repository with Studio. Then Studio will
+automatically parse all required information from `dvc.yaml`, `params.yaml`, and
+other text files that DVC recognizes. The result will be a repository view. The
+view for our project is
+[here](https://studio.iterative.ai/user/alex000kim/views/magnetic-tiles-defect-5kozhnu9jo).
+It displays commits, metrics, parameters, the remote location of data and models
+tracked by DVC, and more.
+
+In the screenshot below, you can see that we created a separate `exp` branch
+that displays the results of the local experiments that we decided to upload to
+our remote repository, like trying different learning rates and batch sizes.
+Note that earlier, we discarded all local experiments whose performance we
+weren't satisfied with.
+
+![Studio view](/uploads/images/2022-05-04/studio_view.png '=800')
+
+To track detailed information about the training process, we integrated
```

Comment:
Should this be a new section? It doesn't seem related to collaboration or Studio.

Reply:
And should it be earlier in the blog post? The example shows results in Studio, but the use case you mention is about diagnosing issues during local training, which would happen before pushing to Studio, right?

Reply:
moved to the earlier section